src/commands/snapshot.ts

```diff
@@ -18,18 +18,18 @@ export default class Snapshot extends PercyCommand {
   static examples = [
     '$ percy snapshot _site/',
     '$ percy snapshot _site/ --base-url "/blog"',
-    '$ percy snapshot _site/ --ignore-files "\.(blog|docs)$"',
+    '$ percy snapshot _site/ --ignore-files "/blog/**,**/blog.*"',
   ]
 
   static flags = {
     'snapshot-files': flags.string({
       char: 'c',
```

Comment:
what do you think about this being `s` instead of `c`?

Reply:
I'm a fan